packages/story-editor/src/components/library/panes/media/local/karma/mediaFetching.karma.js

```diff
@@ -38,42 +37,34 @@ describe('MediaPane fetching', () => {
     fixture.restore();
   });
 
-  it('should fetch 2nd page', async () => {
+  it('should fetch additional pages', async () => {
     const localPane = await waitFor(() =>
       fixture.querySelector('#library-pane-media')
     );
-    const mediaGallery = await within(localPane).getByTestId(
+    const mediaGallery = await within(localPane).findByTestId(
       'media-gallery-container'
     );
 
-    const initialElements =
-      within(mediaGallery).queryAllByTestId(/^mediaElement-/);
+    // ensure fixture.screen has loaded before calling expect to prevent immediate failure
+    const initialElementsLength =
+      fixture.screen.queryAllByTestId(/^mediaElement-/).length;
 
-    await waitFor(() => {
-      // ensure fixture.screen has loaded before calling expect to prevent immediate failure
-      if (initialElements.length !== LOCAL_MEDIA_PER_PAGE) {
-        throw new Error('wait');
-      }
-      expect(initialElements.length).toBe(LOCAL_MEDIA_PER_PAGE);
-    });
+    expect(initialElementsLength).toBeGreaterThanOrEqual(LOCAL_MEDIA_PER_PAGE);
 
-    await mediaGallery.scrollTo(
-      0,
-      mediaGallery.scrollHeight - mediaGallery.clientHeight - ROOT_MARGIN / 2
-    );
+    // Scroll all the way down.
+    await mediaGallery.scrollTo(0, 9999999999999999999);
```

Comment:
As @samwhale mentioned, the viewport for headless versus not headless is different sizes. So scrolling wasn't happening as expected. Now we are just scrolling all the way to the bottom.

Reply:
Were was this mentioned / is this difference documented somewhere?

Reply:
> Were was this mentioned / is this difference documented somewhere?

This hasn't been mentioned or documented anywhere other than right here. This was just discovered, so it would be good to add a comment about this 👍